ament_cmake_python/cmake/ament_python_install_module.cmake

```diff
@@ -17,6 +17,9 @@
 #
 # :param module_file: the Python module file
 # :type MODULE_FILE: string
+# :param DESTINATION: the path to install the module to (default:
+#   PYTHON_INSTALL_DIR)
+# :type DESTINATION: string
```

Comment:
If a value is passed it is still appended to `PYTHON_INSTALL_DIR`. That's not clear from the doc.